### PR TITLE
Feature/admin 142 servant validation

### PR DESCRIPTION
**Pull request by dabick14**

### CODE REVIEW

**Review by jaedag — CHANGES_REQUESTED** [review]

File: api/src/resolvers/resolvers.js [hunk 1]
```diff
@@ -202,6 +203,11 @@ export const MakeServant = async (
   const churchLower = terms.churchLower
   const servantLower = terms.servantLower
 
+  const current_auth_id = splitAuthId(context.jwt.sub)
+  if (Object.values(args).includes(current_auth_id)) {
+    throwErrorMsg('You cannot make yourself a servant!')
+    return
```
[the remark this answers is not in the document]
Comment (dabick14) [reply api/src/resolvers/resolvers.js:209]: It's supposed to make sure that the subsequent parts of the function don't run.

File: api/src/resolvers/resolver-utils.js [hunk 2]
```diff
@@ -364,3 +364,10 @@ export const formatting = (churchType, servantType) => {
     churchLower,
   }
 }
+
+export const splitAuthId = (authID) => {
```
[the remark this answers is not in the document]
Comment (dabick14) [reply api/src/resolvers/resolver-utils.js:368]: Ok

File: web-react/src/pages/dashboards/LogInQueries.js [hunk 3]
```diff
@@ -57,6 +57,7 @@ export const SERVANT_BACENTA_LEADER = gql`
         name
         stream_name
         memberCount
+        vacationStatus
```
[the remark this answers is not in the document]
Comment (dabick14) [reply web-react/src/pages/dashboards/LogInQueries.js:60]: I'm not sure why this seems to have changed. Didn't work in this file

File: web-react/src/pages/dashboards/LogMeIn.jsx [hunk 4]
```diff
@@ -96,7 +96,8 @@ const useLogMeIn = (memberId) => {
   }
 
   const getMember = (response, verb, level) => {
-    return response.data.members[0][`${verb + level}`]
+    const member = response?.data?.members?.length
```
[the remark this answers is not in the document]
Comment (dabick14) [reply web-react/src/pages/dashboards/LogMeIn.jsx:99]: I've pulled from the original deploy. Hope it solves the issue

**Comment by jaedag** [comment]
Hi. Just reminding that we are waiting for your changes to merge this PR @dabick14

**Comment by jaedag** [comment]
By this point you should probably close this PR and redo your changes in a new branch off of deploy

**Comment by jaedag** [comment]
What about this one?

**Comment by dabick14** [comment]
Made the changes. Will pull from deploy and submit another request

**Pull request closed without merging**